app/ui-react/packages/ui/src/Customization/apiClientConnectors/ApiConnectorReview.tsx

```diff
@@ -37,113 +38,115 @@ export class ApiConnectorReview extends React.Component<
-                    {this.props.apiConnectorName}
-                  </TextListItem>
-                  <TextListItem component={TextListItemVariants.dt}>
-                    {this.props.i18nDescriptionLabel}
-                  </TextListItem>
-                  <TextListItem component={TextListItemVariants.dd}>
-                    {this.props.apiConnectorDescription}
-                  </TextListItem>
-                </TextList>
-              </Container>
-              <Title
-                headingLevel="h5"
-                size="md"
-                className="customization-details__heading"
-              >
-                {this.props.i18nImportedHeading}
-              </Title>
-              <Container>
-                <Text
-                  component={TextVariants.p}
-                  dangerouslySetInnerHTML={{
-                    __html: this.props.i18nOperationsHtmlMessage,
-                  }}
-                />
-              </Container>
-
-              {/* tagged messages */}
-              {this.props.i18nOperationTagHtmlMessages && (
-                <TextList className="api-connector-review__tagMessageList">
-                  {this.props.i18nOperationTagHtmlMessages.map(
-                    (msg: string, index: number) => (
-                      <TextListItem
-                        key={index}
-                        dangerouslySetInnerHTML={{ __html: msg }}
-                      />
-                    )
-                  )}
-                </TextList>
-              )}
-
-              {/* error messages */}
-              {this.props.i18nErrorsHeading && this.props.errorMessages && (
+      <PageSection>
+        <Card>
+          <CardHeader>
+            <Title size="lg">{this.props.i18nTitle}</Title>
+          </CardHeader>
+          <CardBody>
+            {this.props.i18nValidationFallbackMessage ? (
+              <h5 className="api-connector-review__validationFallbackMessage">
+                {this.props.i18nValidationFallbackMessage}
+              </h5>
+            ) : (
+              <TextContent>
                 <Title
                   headingLevel="h5"
                   size="md"
                   className="customization-details__heading"
                 >
-                  {this.props.i18nErrorsHeading}
+                  {this.props.i18nApiDefinitionHeading}
                 </Title>
-              )}
-              {this.props.errorMessages
-                ? this.props.errorMessages.map(
-                    (errorMsg: string, index: number) => (
-                      <Text component={TextVariants.p} key={index}>
-                        {index + 1}. {errorMsg}
-                      </Text>
-                    )
-                  )
-                : null}
-
-              {/* warning messages */}
-              {this.props.i18nWarningsHeading && this.props.warningMessages && (
+                <Container>
```

Comment:
I think we want to try avoiding `Container`, no?  I thought this was mostly to work around integrating pf4 things into pf3 cards (@seanforyou23 correct me if I'm wrong), might be work fine to just remove this.